lightly_studio/src/lightly_studio/api/routes/api/sample.py

```diff
@@ -24,23 +24,37 @@
 )
 from lightly_studio.resolvers import (
     image_resolver,
+    sample_resolver,
     tag_resolver,
 )
-from lightly_studio.resolvers.image_resolver import GetAllSamplesByDatasetIdResult
 from lightly_studio.resolvers.samples_filter import (
     SampleFilter,
 )
 
 samples_router = APIRouter(prefix="/datasets/{dataset_id}", tags=["samples"])
 
 
-@samples_router.post("/samples", response_model=ImageView)
+@samples_router.post("/samples")
 def create_sample(
     session: SessionDep,
     input_sample: ImageCreate,
-) -> ImageTable:
+) -> ImageView:
```

Comment:
Need to manually convert the output since ImageTable is not yet compatible with ImageView (I did not want to change the API interface).

Reply:
I like the idea of creating new routes for image fetching